packages/ui/src/pages/Home/MultisigActionMenu.tsx

```diff
@@ -45,30 +49,48 @@ const MultisigActionMenu = ({ withSendButton = true }: MultisigActionMenuProps)
         onClick: () => setIsChangeMultiModalOpen(true)
       })
 
+    // If we are NOT rendering "new transaction button", the functionality will appear in the list
+    if (!withNewTransactionButton) {
+      opts.unshift({
+        text: 'Send transaction',
+        icon: <HiOutlinePaperAirplane size={20} />,
+        onClick: () => setIsSendModalOpen(true)
+      })
+    }
+
```

Comment:
This is not needed (for now at least) 
```suggestion
```

Reply:
Yeah, it's required only when we have a separate button for each multisig. I will comment this line